code/obj/item/clothing/chameleon_js.dm

```diff
@@ -269,3 +272,1659 @@
 		desc = "This suit is ludicrously cheap. They must be embezzling the research budget again."
 		icon_state = "director"
 		item_state = "director"
+
+	rank/medical_director
+		name = "medical director's uniform"
+		desc = "There's some odd stains on this thing. Hm."
+		icon_state = "med_director"
+		item_state = "med_director"
+
+	rank/security_assistant
+		name = "security assistant uniform"
+		desc = "Wait, is that velcro?"
+		icon_state = "security-assistant"
+		item_state = "security-assistant"
+
+	rank/rancher
+		name = "rancher's overalls"
+		desc = "Smells like a barn; hopefully its wearer wasn't raised in one."
+		icon_state = "rancher"
+		item_state = "rancher"
+
+/obj/item/clothing/head/chameleon
+	name = "hat"
+	desc = "A knit cap in red."
+	icon_state = "red"
+	item_state = "swat_gl"
```

Comment:
this `item_state` seems wrong. should probably be `rgloves`?
wait why do beanies use glove item_states?????